micronaut-projects/micronaut-build
# Docs: Improved look-and-feel for mobile devices

These changes are from this PR: https://github.com/micronaut-projects/micronaut-docs/pull/81, which I closed because I found that resources in that repo are used only for micronaut docs prior to 2.5.x.

Please check that PR for more details about visual changes.

How to test these changes?

I'm using `micronaut-security` as an example.

1. Update version in `settings.gradle`:

```groovy
plugins {
    id("io.micronaut.build.shared.settings") version "5.3.16-SNAPSHOT"
}
```

2. Build docs:

```bash
gradle clean zipDocs
```

3. Unzip and serve static files:

```bash
cd build/distributions/
unzip security-parent-docs-3.7.0-SNAPSHOT.zip
```

Using Node.js `http-server` (https://www.npmjs.com/package/http-server) to start a web server:

```bash
npm install --global http-server

http-server -c-1
```

I use a mobile device to check (in my case I'm checking on iPhone 13 Pro Max), but also can use Google Chrome with emulating mobile phone: https://developer.chrome.com/docs/devtools/device-mode/
